docs/contribution/devEnv.md

```diff
@@ -236,7 +240,7 @@ SkillTree's overall testing strategy is to implement black-box integration tests
 
 We like tests (especially integration tests), so please make sure you thoroughly test your code.  
 Prior to making a Pull Request make sure that ALL tests pass:
-1. Run services tests against PostgreSQL in addition to the default in-memory H2 database
+1. Run all services tests
```

Comment:
Might want to change `services` to `service`

Reply:
:+1: